src/__test__/components/__snapshots__/BCHeader.test.tsx.snap

```diff
@@ -10,21 +10,27 @@ exports[`the Header component should match the snapshot 1`] = `
     data-testid="header"
   >
     <a
-      className="cds--header__name"
+      className="header-link"
       data-testid="header-name"
-      href="/"
+      href="/home"
+      onClick={[Function]}
     >
-      <span
-        className="cds--header__name--prefix"
-      >
-        BC Gov's
-      </span>
-       
-      NR Sample App
+      BC Gov's NR Sample App
     </a>
     <div
       className="cds--header__global"
     >
+      <button
+        aria-describedby={null}
+        aria-pressed={null}
+        className="cds--btn cds--btn--sm cds--btn--primary"
```

Comment:
I remember @mgaseta commenting that he changed the prefix to `bcgov`. I would suggest you take note of it and open a task to review it after double check with him.

Reply:
I think the prefix is going to be applied after. If you look at the`src/index.tsx` file you'll see a `ClassPrefix` with `bcgov` on it!